libs/mailroom/ngx/src/lib/modules/list/pages/detail/detail.component.scss

```diff
@@ -0,0 +1,59 @@
+.attachment {
+  height: 100%;
+  width: 100%;
+}
+
+.to-inbox {
+  display: inline-flex;
+  align-items: flex-start;
+  margin-left: 5%;
+  margin-bottom: 16px;
+  margin-top: 10px;
+  cursor: pointer;
+}
+
+.info-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.attachments-container {
+  display: flex;
+  flex-direction: row;
+}
+
+#attachment-header {
+  padding-top: 16px;
+}
+
+.container {
+  padding-left: 16px;
+  padding-right: 16px;
+  display: flex;
+  flex-direction: column;
+  padding-left: 20%;
+  padding-right: 20%;
+}
+
+.column-thing {
+  column-count: 2;
+  padding-inline-start: 0;
+  li {
+    list-style: none;
+  }
+}
+
+.attachment-image {
+  width: 100%;
+  height: 30rem;
+  background-size: contain;
+  object-fit: contain;
+}
```

Comment:
Need to add `background-repeat: no-repeat` else smaller attachments will repeat

![image](https://user-images.githubusercontent.com/3969818/180100728-f2a99437-d5b5-4dc7-bfd4-3530cbe7db56.png)

Reply:
Fixed.